src/main/java/com/adobe/target/edge/client/service/DefaultTargetService.java

```diff
@@ -53,8 +55,22 @@ public DefaultTargetService(ClientConfig clientConfig, TelemetryService telemetr
     } else {
       this.targetHttpClient = targetHttpClient;
     }
+
+    Properties defaultProps = new Properties();
```

Comment:
Let's extract this whole section to a separate `loadDefaultProperties` method

Reply:
I believe `this.SDK_VERSION` can no longer being a final var if we do this. Should be okay though as it is not referenced anywhere else